### ISSUE TITLE
1.1.0 getOperationsList issue

### ISSUE THREAD

**Issue by dominikkarall** [lendy007/skodaconnect#37]
Hi,

unfortunately I get the following error with 1.1.0
```
2021-09-04 09:09:06,434 - ERROR    - skodaconnect.connection: Got HTTP 400 400, message='Bad Request', url=URL('https://msg.volkswagen.de/api/rolesrights/operationlist/v3/vehicles/xxxxxxxxxx')"Bad Request" from server, this request might be malformed or not implemented correctly for this vehicle
```

I couldn't get the root cause of this one, maybe it has something to do with the new token handling?

**Comment by Farfar** [comment]
> Hi,
> 
> unfortunately I get the following error with 1.1.0
> 
> ```
> 2021-09-04 09:09:06,434 - ERROR    - skodaconnect.connection: Got HTTP 400 400, message='Bad Request', url=URL('https://msg.volkswagen.de/api/rolesrights/operationlist/v3/vehicles/xxxxxxxxxx')"Bad Request" from server, this request might be malformed or not implemented correctly for this vehicle
> ```
> 
> I couldn't get the root cause of this one, maybe it has something to do with the new token handling?

Yes it seems like some have trouble with this, what car model do you have?
There has been changes to some HTTP headers when authorizing client and fetching data but so far I have only had one report where it has been an issue involving Skoda Citigo.
I myself have a Superb iV and everything is working as expected.
I will backtrace and fix this but need to know of this is an issue for certain models or only certain regions/accounts.

**Comment by Farfar** [comment]
Related to #36

**Comment by jfmennedy** [comment]
Hi, the car related to this problem is a Skoda Octavia RS 2019 model with active subscription to skoda online services.

**Comment by Farfar** [comment]
> Hi, the car related to this problem is a Skoda Octavia RS 2019 model with active subscription to skoda online services.

Which was latest working release?

**Comment by jfmennedy** [comment]
latest working release was 1.0.42

**Comment by Farfar** [comment]
> latest working release was 1.0.42

Can you try removing the comment on line 29 of skodaconnect/const.py and then commenting out line 30?
So instead of:
```
#XCLIENT_ID = '28cd30c6-dee7-4529-a0e6-b1e07ff90b79'
XCLIENT_ID = 'a83d7e44-c8b7-42b7-b8ca-e478270d2091'
```
You have:
```
XCLIENT_ID = '28cd30c6-dee7-4529-a0e6-b1e07ff90b79'
#XCLIENT_ID = 'a83d7e44-c8b7-42b7-b8ca-e478270d2091'
```

**Comment by jfmennedy** [comment]
ok, i tried this but still same error.

**Comment by Farfar** [comment]
> ok, i tried this but still same error.

I think I've found the problem.
On line 767, change from:
```
                    self._session_auth_ref_url,
```
To:
```
                    self._session_spin_ref_url,
```

**Comment by jfmennedy** [comment]
with this modifiaction it works, even for both clientid

**Comment by Farfar** [comment]
> with this modifiaction it works, even for both clientid

Great! Will fix asap.

**Comment by Farfar** [comment]
Fixed in latest commit, scheduled for 1.1.1 release.
